src/main/java/com/google/sps/servlets/FormHandlerServlet.java

```diff
@@ -76,10 +97,23 @@ public void doPost(HttpServletRequest request, HttpServletResponse response)
     //TODO: Sentiment analysis for textbox
 
     //Creating a post class
-    Post newPost = new Post(locationName, category, parking, ratingScore, noiseScore, spaceScore, userReview, imageURL, hasImage);
+    Post newPost = new Post(locationName, category, parking, ratingScore, noiseScore, spaceScore, userReview, imageURL, hasImage,  imageTags);
 
     //Printing data to confirm that form contents have been read
-    String form = "Location: " + "[" + locationName + "]" + ", Category: " + category + ", Parking Available: " + parking + ", Overall Rating: "+ ratingScore + ", Noise Rating: " + noiseScore + ", Space Rating: " + spaceScore  + ", Image Uploaded: " + hasImage + ", Image URL: " + "[" + imageURL + "]" + ", User Review: " + "[" + userReview +"]";
+    String form = "Location: " + "[" + locationName + "]" + ", Category: " + category + ", Parking Available: " + parking + ", Overall Rating: "+ ratingScore + ", Noise Rating: " + noiseScore + ", Space Rating: " + spaceScore + ", User Review: " + "[" + userReview +"]"  + ", Image Uploaded: " + hasImage + ", Image URL: " + "[" + imageURL + "]";
```

Comment:
I think it'd be prettier to have all this as a separate function. maybe like generateDebugString(...) or something.